### PR TITLE
Don't use process.exit, but set exit code

### PR DESCRIPTION
**Pull request by LeMisterV**
Hi,

I had some problems with process exiting before stdout write is finished.
I got this when using json-cov mocha reporter, which is dumping lots of data to stdout when there's lots of tests and many lines of code.

With this commit, the mocha-phantomjs executable will no longer force process to exit quickly. It will wait for the process to exit naturally, and the expected exit code will be used.

This behavior is recommended in nodejs documentation : 
https://nodejs.org/api/process.html#process_process_exit_code

I also changed exit codes for positive codes 

Hope you like it ;)

### CODE REVIEW

**Comment by nathanboktae** [comment]
I like this! but I don't understand why you are changing the exit code numbers - that will constitute a breaking change to go in a major version.

Restore the codes and I'll be glad to take this, thanks :)

**Comment by LeMisterV** [comment]
Just pushed back original error codes

File: bin/mocha-phantomjs [hunk 1]
```diff
@@ -159,12 +160,17 @@ try {
       if (phantomjs.signalCode !== null) {
         console.log("phantomjs terminated with signal " + phantomjs.signalCode);
       }
-      code = -1;
+      code = 1;
     } else if (code === 127) {
-      console.log("Perhaps phantomjs is not installed?");
+      console.log("Perhaps phantomjs is not installed ?");
```
Comment (nathanboktae) [inline bin/mocha-phantomjs]: this is really random and not standard English....

**Comment by nathanboktae** [comment]
Thanks!

**Pull request merged**